src/connections.c

```diff
@@ -1920,7 +1920,7 @@ connection_handle_read_post_state (request_st * const r)
         chunkqueue_remove_finished_chunks(cq);
     }
 
-    if (dst_cq->bytes_in == (off_t)r->reqbody_length) {
```

Comment:
At first glance, this seems incorrect since it no longer handles a chunked request body that turned out to be empty, ending with a `0\r\n\r\n` chunk with a 0-length body.